assembly/JSON.ts

```diff
@@ -201,7 +201,7 @@ export class Str extends Value {
   }
 
   toString(): string {
-    return this._str;
+    return `"${this._str}"`;
```

Comment:
Hmmmm, so the reason why I removed the quotes here, is because if you get a JSON string, it's extremely hard to tell when you log the string if the quotes are a part of the string, or just the output of the string. 

Could we perhaps add a new function like: `toJSONString()`, that way we can signify it is a string that will work in JSON, vs. the string representation of the value? I don't think that would break be a breaking change, but could allow us to fix this internally. Unless you at NEAR have a bunch of code using .toString directly expecting the quotes? 🤔